### PR TITLE
test badge bruh

### PR DESCRIPTION
**Pull request by chetanddesai**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java [hunk 1]
```diff
@@ -195,16 +195,18 @@ public StatusImage getTestResultImage(Integer testPass, Integer testTotal) {
      * @return
      */
     private String replaceTestResultSVG(String image, Integer testPass, Integer testTotal) {
+    	
+    	int passPercent = (testPass / testTotal) * 100;
```
Comment (sputnikci) [inline src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java:199]: [FindBugs] WARNING: RCN: Nullcheck of testTotal at line 201 of value previously dereferenced in org.jenkinsci.plugins.badge.ImageResolver.replaceTestResultSVG(String, Integer, Integer)

File: src/main/java/org/jenkinsci/plugins/badge/PublicBadgeAction.java [hunk 2]
```diff
@@ -166,8 +166,14 @@ public HttpResponse doTestIcon(StaplerRequest req, StaplerResponse rsp, @QueryPa
...
 
         if (project.getLastSuccessfulBuild() != null) {
-            testPass = 1;
-            testTotal = 2;
+        	AbstractTestResultAction testAction =  project.getLastSuccessfulBuild().getAction(AbstractTestResultAction.class);
+			if(testAction != null){
+				int total = testAction.getTotalCount();
+				int pass = total - testAction.getFailCount();
+				
+				testTotal = new Integer(total);
```
Comment (sputnikci) [inline src/main/java/org/jenkinsci/plugins/badge/PublicBadgeAction.java:174]: [FindBugs] INFO: Bx: org.jenkinsci.plugins.badge.PublicBadgeAction.doTestIcon(StaplerRequest, StaplerResponse, String) invokes inefficient new Integer(int) constructor; use Integer.valueOf(int) instead

File: src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java (hunk 1 above)
Comment (chetanddesai) [inline src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java:199]: dammit

Comment (chetanddesai) [inline src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java:199]: I'm trying to hack here. Not be a software engineer.

Comment (chetanddesai) [inline src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java:199]: asshole..

Comment (SxMShaDoW) [inline src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java:199]: can we get this installed on build jobs as well when we move things to java 👍

**Pull request merged**